index.html

```diff
@@ -5,6 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <div class="navbar__logo">Ahmed</div>
+      <div class="navbar__hamburger">
+        <img src="images/Union.svg" alt="menu" />
+      </div>
+    </nav>
```

Comment:
Hi @AhmedHossam01, I can see you are doing a great job on this project 😺. 

However for the toolbar, I think you still have a few changes to make 🤔 

- [ ] The container of the section is meant to be wrapped in a `<header>` tag. The header usually contains branding information of a website and the navigation.
- [ ] The logo is also meant to be a link (a tag). This is because is usually clicked on by users to revert back to the homepage. So it would really be nice if it was clickable 😄.
- [ ] The hamburger button has a `<nav>` tag and I think it would be nice if we followed this structure 👇🏻 

```
<header>
  <a href=""> -- logo goes here -- </a>
  <nav>
    <button> -- menu image goes here -- </button>
  </nav>
</header>
```
❓ What do you think about this structure?